trio/tests/test_socket.py

```diff
@@ -101,11 +101,19 @@ def test_socket_has_some_reexports():
 async def test_getaddrinfo(monkeygai):
     def check(got, expected):
         # win32 returns 0 for the proto field
-        def without_proto(gai_tup):
-            return gai_tup[:2] + (0,) + gai_tup[3:]
+        # alpine and glibc have inconsistent handling of the canonical name
```

Comment:
This should probably be the following:

```suggestion
        # musl and glibc have inconsistent handling of the canonical name
```

If libc is the issue, then Musl is causing it, not Alpine :wink:

Reply:
Fair enough :-)